### DISCUSSION TITLE
Welcome to tswg-tsp-specification Discussions!

### DISCUSSION THREAD

**Discussion by m00sey** [trustoverip/tswg-tsp-specification#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by talltree** [comment]
This is a special welcome to any reviewers/commenters on our Implementers Draft. In case you arrived here from another source, [here is the ToIP Foundation blog post announcing the first Implementers Draft](https://trustoverip.org/blog/2024/04/11/toip-announces-the-first-implementers-draft-of-the-trust-spanning-protocol-specification/#howdiffer). We'll also be holding several sessions covering this draft at next week's [Internet Identity Workshop](http://www.internetidentityworkshop.com/).

We welcome all questions, comments, or discussions about the TSP.
